### ISSUE TITLE
Simpler way to use spv4

### ISSUE THREAD

**Issue by DuzzuAcid** [iwalton3/jellyfin-mpv-shim#141]
I would like to use this great piece of software with svp4 as it's beautiful when it fills the black bars on movies and I just bought some led strips and it would be wonderful to use them just clicking play on a movie in jellyfin.

I know there already is a way and it's written in #svp-integration but sadly I don't really understand how to configure everything to make the two softwares work together

Would it be possible to have an executable that configures everything needed or at least a step by step description that could take me by the hand to the goal I'm trying to reach?

Thank you very much for what you did here, I really appreciate your work every day.

**Comment by iwalton3** [comment]
If you need help understanding a specific part of the instructions, please let me know. I’m also willing to accept improved documentation.

I’m assuming you are on Windows.

**Comment by DuzzuAcid** [comment]
Yep, I think I need help understanding how to make it work, I'm on windows 20H2, maybe the problem is the platform I'm on because it's a Celeron N3350, but I don't need svp4 to smooth the movies to 60fps, I just need it to fill the black bars and use the rgb leds on the back of the tv so it shouldn't really be a problem in terms of raw power as it already works with mpc-hc reading hevc and also can convert the frame rate to 60fps in 1080p movies.

So, I have personal mpv on C:\MPV folder where I extracted in order:
- mpv-x86_64-20201220-git-dde0189.7z
- VapourSynth64-Portable-R52.7z
- python-3.8.6-embed-amd64.zip
- vapoursynth-mvtools-v23-win64.7z

I enabled External control via HTTP on svp4 on default port 9901 and set up firewall to not block connections on that port, don't know if it's needed but I did anyway.

Last thing, these are my two configuration files conf.json and mpv.conf, taken from "%appdata%\Roaming\jellyfin-mpv-shim"
[conf.json&mpv.conf.zip](https://github.com/iwalton3/jellyfin-mpv-shim/files/5734194/conf.json.mpv.conf.zip)

When I start Jellyfin mpv desktop everything seems good, I can see every list, but when I click on a movie and it starts mpv it hangs for 3 seconds and then force closes both mpv player and the jellyfin browser, starting as administrator is different as mpv lets me see movies but.. svp4 doesn't see mpv when I start your jellyfin as administrator so, what am I doing wrong?

**Comment by iwalton3** [comment]
Replace
```
    "mpv_ext_path": C:\\MPV\\mpv.exe,
    "svp_socket": mpvpipe,
```
with
```
    "mpv_ext_path": "C:\\MPV\\mpv.exe",
    "svp_socket": "mpvpipe",
```

**Comment by DuzzuAcid** [comment]
Aaaaaand it works, I can't believe it was that simple, it just needed apexes, well, thank You, I didn't know I had to input those settings this way as I couldn't see apexes in here: https://github.com/iwalton3/jellyfin-mpv-shim#svp-integration
